### DISCUSSION TITLE
1 year long time-lapse! and first problems

### DISCUSSION THREAD

**Discussion by Zampa1985** [gopro/labs#991] [Q&A]
Hi everyone,
I'm an Italian professional photographer and I was commissioned by a National park to make a 1 year time-lapse to see the change of seasons on a mountain view and I thought to do it with a GoPro. 
The GoPro will be outside, mounted on a pole equipped with electricity. I will put it on a waterproof case with a "watertight hole" for the usb cable. The cable will be connected to a charger/power supply.
I bought a GoPro 10 black and installed Labs.
Yesterday I started a 1 week test inside my home, just to be sure everything worked well, but this morning I found the GoPro crashed. On the rear monitor there were the shutdown screen, with the spinning icon.
Do you think it's an electric problem? Is maybe the charger is not compatible?
Both the charger and the GoPro were hot. I used a normal mobile charger, output 5.0V, 2.1A.

Do you think I'm asking too much to my GoPro? Someone tried to do a thing like this?

Thank you so much, 
Andrea

**Comment by dnewman-gpsw** [comment]
I would only recommended a HERO12 for very long form timelapse. I've had a HERO12 running a timelapse since December (although different timelapses) that camera is simple much more reliable. 

Using HERO10, it would only if you use time lapse video recording continuously, no shutdowns or restarts (as this is the most common failure point), no internal battery, so your external power will have to be reliable.  There currently is a Labs bug that adds to the HERO10 endurance issues if using Night Lapse Photo.

**Comment by Espusen** [comment]
Agree with David above.
I have also tested HERO 12 extensively on long form timelapse with great success. All my tests with Hero 9 and 10 failed.

Look me up [here](www.altofocus.co.uk) if you need more help! I have a library with loads of codes editing help and also weatherproof housings with solar panel for the GoPro.
I have helped the National Trust here in the uk on similar projects. One of my clients filmed an oak tree with our kit for 1 whole year.
This was done with HERO 12, Labs and solar power alone.

Espen

**Reply by Zampa1985** [reply]
Hi Espen, the link you gave me doesn't work. Can you control it and maybe resend it to me? :-) 

I think I will disturb you again for some more help in the next days because I'm so ignorant about GoPro and time-lapse world...

Thank you so much,
Andrea

**Reply by Espusen** [reply]
No problem Andrea. Please ask away.
Website is: www.AltoFocus.co.uk
You can access the Labs resources here: https://www.altofocus.co.uk/gopro-labs
£20 for lifetime access.

**Comment by Zampa1985** [comment]
Hi guys, thank you so much for your answers.
Today I bought a HERO 12 after your advice.
In these days I will try to understand something about Labs and I will sure come back to you for more helps!

Thanks,
Andrea
